modules/mockk-dsl/src/commonMain/kotlin/io/mockk/API.kt

```diff
@@ -460,13 +460,13 @@ object MockKDsl {
     /**
      * Static mockk
      */
-    inline fun internalMockkStatic(vararg classes: KClass<*>) {
+    inline fun internalMockkStatic(classes: Array<out KClass<*>>) {
         val factory = MockKGateway.implementation().staticMockFactory
 
         classes.forEach {
             val cancellation = factory.staticMockk(it)
 
-            internalClearStaticMockk(it)
+            internalClearStaticMockk(arrayOf(it))
```

Comment:
It seems more efficient to call `internalClearStaticMockk` together as follows, but why not?
If there are no problems, I will reflect the changes.

```kotlin
classes.forEach {
    val cancellation = factory.constructorMockk(it, recordPrivateCalls, localToThread)

    MockKCancellationRegistry
        .subRegistry(MockKCancellationRegistry.Type.CONSTRUCTOR)
        .cancelPut(it, cancellation)
}

internalClearConstructorMockk(classes)
```